src/lang/english.txt

```diff
@@ -5045,13 +5045,13 @@ STR_FORMAT_INDUSTRY_NAME                                        :{TOWN} {STRING}
 STR_FORMAT_WAYPOINT_NAME                                        :{TOWN} Waypoint
 STR_FORMAT_WAYPOINT_NAME_SERIAL                                 :{TOWN} Waypoint #{COMMA}
 
-STR_FORMAT_DEPOT_NAME_TRAIN                                     :{TOWN} Train Depot
-STR_FORMAT_DEPOT_NAME_TRAIN_SERIAL                              :{TOWN} Train Depot #{COMMA}
-STR_FORMAT_DEPOT_NAME_ROAD_VEHICLE                              :{TOWN} Road Vehicle Depot
-STR_FORMAT_DEPOT_NAME_ROAD_VEHICLE_SERIAL                       :{TOWN} Road Vehicle Depot #{COMMA}
-STR_FORMAT_DEPOT_NAME_SHIP                                      :{TOWN} Ship Depot
-STR_FORMAT_DEPOT_NAME_SHIP_SERIAL                               :{TOWN} Ship Depot #{COMMA}
-STR_FORMAT_DEPOT_NAME_AIRCRAFT                                  :{STATION} Hangar
+STR_FORMAT_DEPOT_NAME_TRAIN                                     :{TOWN} Train depot
```

Comment:
There are in the middle of a sentence, so should be all-lowercase.

Reply:
Can show me a screenshot example?

Reply:
![image](https://user-images.githubusercontent.com/37945304/81870918-71eaa180-953c-11ea-8fde-4d5a3ac7d92c.png)

Reply:
Thanks. Then all of these have to be changed for consistency, including the map legend names. Come to think of it, that was actually part of the rule I wrote for title capitalization, and now I remember why I did it like that. Will fix soon.

Reply:
> Come to think of it, that was actually part of the rule **I** wrote for title capitalization

By the way, are you "‎Mcbanhas" on the wiki?

Reply:
Yes!